packages/nextjs/src/config/define-cli-config.ts

```diff
@@ -33,3 +35,17 @@ function addDefaultScaffoldTemplates(cliConfig: SitecoreCliConfigInput) {
 
   cliConfig.scaffold.templates.unshift(defaultTemplate, byocTemplate);
 }
+
+// /**
```

Comment:
This is a temporary measure - uncommenting this will cause nextjs app to fail at build time. `/config` submodule would try importing 'fs' module via `generateMap` and that will fail when nextjs generates client bundle and pages.
The solution is to separate `config` submodule into `config` and `cliConfig` - the latter being able to use server-side logic. This is in progress, but I didn't want this to block the PR, to get comments on overall implementation